### PR TITLE
Updated how-to bridge docs

### PR DESCRIPTION
**Pull request by degengami**
Added in Moonbeam Bridge how-to and updated Acala Bridge how-to

### CODE REVIEW

File: docs/howto_bridge.md [hunk 1]
```diff
@@ -5,123 +5,104 @@ title: Bridge Assets
...
 
-Once the transaction has been processed by Wormhole, click Redeem (9). This action results in you receiving the tokens on the Ethereum chain.
+### Step 3: Bridge Tokens
 
-<div style={{textAlign: 'center'}}>
-  <img alt="metadata" src={useBaseUrl('/howto_bridge/outbound_5.jpg')} width="800px" />
-</div>
+- Enter the amount of tokens and the token for bridging **(4)**.
+- To commence the bridge, click **Approve Tokens (5)** and sign the transaction using your Metamask wallet app.
+- Once the tokens are approved for transfer, click **Send Tokens (5). T**his starts the bridging process cross-chain.
```
Comment (lolmcshizz) [inline docs/howto_bridge.md:105]: T**his? ** in wrong place

File: docs/howto_bridge.md [hunk 2]
```diff
@@ -5,123 +5,104 @@ title: Bridge Assets
...
+- Once the transaction has been processed by Wormhole, click **Redeem Tokens (5).** This action results in you receiving the tokens on the Acala chain.
```
Comment (lolmcshizz) [inline docs/howto_bridge.md:106]: You still have to "redeem" the tokens? This doesn't use an auto-relayer? What does the user do once they're on the Acala chain? The UI should abstract the next step out for them to appear on HydraDX?

File: docs/howto_bridge.md [hunk 3]
```diff
@@ -5,123 +5,104 @@ title: Bridge Assets
...
+
+
+
+## To/From Ethereum via Acala {#acala}
+
+### Prerequisites
+
+- A Polkadot wallet (Talisman or Polkadot.js/apps);
+- An Ethereum wallet (Metamask);
+- Bind your two wallets following [Acala's guide](https://guide.acalaapps.wiki/general/address-binding). Completing this action will require a small amount ACA.
```
Comment (lolmcshizz) [inline docs/howto_bridge.md:79]: small amount of ACA

File: docs/howto_bridge.md (hunk 2 above)
Comment (lolmcshizz) [reply docs/howto_bridge.md:106]: Okay it seems it should be "Redeem & route tokens" - this action results in you receiving the tokens on the HydraDX chain.

File: docs/howto_bridge.md [hunk 4]
```diff
@@ -5,123 +5,104 @@ title: Bridge Assets
...
-You are all set!
+- Select the token asset and amount of tokens you would like to bridge **(4)**.
 
-## Bridging Assets Acala -> ETH
-After transferring bridged assets back to Acala (using [cross-chain transfers](/howto_xcm)), you can bridge them back to the Ethereum ecosystem following
+### 05 Bridge Tokens
 
-### 01 Navigate to Wormhole Token Bridge Page
-[https://www.portalbridge.com/#/transfer](https://www.portalbridge.com/#/transfer)
+- Within **Settings** **(5)**, you can select whether to Auto Relay the transaction.
```
Comment (lolmcshizz) [inline docs/howto_bridge.md:61]: Shouldn't this step be urging the user to select this? otherwise I'm guessing they won't be auto routed to HydraDX?

**Review by lolmcshizz — CHANGES_REQUESTED** [review]

**Review by degengami — COMMENTED** [review]
revised

**Review by lolmcshizz — APPROVED** [review]

**Pull request merged**